### PR TITLE
Venu/containerization

### PR DESCRIPTION
**Pull request by v-vgujjula**
containerization integration for data services

### CODE REVIEW

File: testsuite/launch.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@ spec:
       containers:
       - name: azure-arc-kubernetes-conformance
         # test image will change to actual once we push to MCR
-        image: testsantoshacr.azurecr.io/arck8sconformance/launcher:0.0.37
+        image: testsantoshacr.azurecr.io/arck8sconformance/launcher:0.0.78
```
Comment (anraghun) [inline testsuite/launch.yaml:40]: Why is this change required?

**Review by anraghun — CHANGES_REQUESTED** [review]
Won't you need to capture the image upgrade for the az cli changes you've made to both directly and indirectly connected mode plugins?

File: testsuite/launch.yaml (hunk 1 above)
Comment (santosh02iiit) [reply testsuite/launch.yaml:40]: the container version 0.0.78 is having the latest change of launcher container code. previous versions are giving error. Eventually we have to change this ACR path to MCR path when we push the docker image.

**Review by anraghun — APPROVED** [review]
